pkg/cli/environment.go

```diff
@@ -109,6 +113,8 @@ func New() *EnvSettings {
 		RepositoryCache:           envOr("HELM_REPOSITORY_CACHE", helmpath.CachePath("repository")),
 		BurstLimit:                envIntOr("HELM_BURST_LIMIT", defaultBurstLimit),
 		QPS:                       envFloat32Or("HELM_QPS", defaultQPS),
+		MaxChartSize:              envInt64Or("HELM_MAX_CHART_SIZE", 0),
```

Comment:
```suggestion
		MaxChartSize:              envInt64Or("HELM_MAX_CHART_SIZE", 100 * 1024 * 1024), // Default 100 MiB
```

Lets put the defaults into the defaults :) 

Then `helm env` can print them, and users don't have to go hunt for the "actual default".